src/lib.rs

```diff
@@ -46,6 +49,14 @@ pub type FxHashMap<K, V> = HashMap<K, V, BuildHasherDefault<FxHasher>>;
 #[cfg(feature = "std")]
 pub type FxHashSet<V> = HashSet<V, BuildHasherDefault<FxHasher>>;
 
+/// Type alias for a hashmap using the `fx` hash algorithm with [`FxRandomState`].
+#[cfg(feature = "rand")]
+pub type FxHashMapRnd<K, V> = HashMap<K, V, FxRandomState>;
```

Comment:
```suggestion
pub type FxHashMapRand<K, V> = HashMap<K, V, FxRandomState>;
```
I'd prefer writing out the `a`, or maybe dropping these aliases alltogether.

Reply:
🆗